lib/ios/RNNComponentPresenter.m

```diff
@@ -92,72 +92,72 @@ - (void)applyOptionsOnInit:(RNNNavigationOptions *)options {
 
 - (void)mergeOptions:(RNNNavigationOptions *)options resolvedOptions:(RNNNavigationOptions *)currentOptions {
     [super mergeOptions:options resolvedOptions:currentOptions];
-	RNNNavigationOptions * withDefault	= (RNNNavigationOptions *) [[currentOptions overrideOptions:options] withDefault:[self defaultOptions]];
-	UIViewController* viewController = self.boundViewController;
+    RNNNavigationOptions * withDefault    = (RNNNavigationOptions *) [[currentOptions overrideOptions:options] withDefault:[self defaultOptions]];
+    UIViewController* viewController = self.boundViewController;
 
-	if (options.backgroundImage.hasValue) {
-		[viewController setBackgroundImage:options.backgroundImage.get];
-	}
-	
-	if (options.topBar.searchBar.hasValue) {
-		BOOL hideNavBarOnFocusSearchBar = YES;
-		if (options.topBar.hideNavBarOnFocusSearchBar.hasValue) {
-			hideNavBarOnFocusSearchBar = options.topBar.hideNavBarOnFocusSearchBar.get;
-		}
-		[viewController setSearchBarWithPlaceholder:[options.topBar.searchBarPlaceholder getWithDefaultValue:@""] hideNavBarOnFocusSearchBar:hideNavBarOnFocusSearchBar];
-	}
+    if (options.backgroundImage.hasValue) {
+        [viewController setBackgroundImage:options.backgroundImage.get];
+    }
+    
+    if (options.topBar.searchBar.hasValue && options.topBar.searchBar.get) {
```

Comment:
Is it safe to check it this way? Is there a better way to make sure topBar.searchBar option was provided and it's indeed `true`?